src/components/Icon/types.ts

```diff
@@ -5,127 +5,129 @@ import { LogoName } from "../Logos/types";
 import { PaymentName, PaymentProps } from "../icons/Payments";
 
 export type IconSize = "xs" | "sm" | "md" | "lg" | "xl" | "xxl";
-export type IconName =
```

Comment:
Just out of interest, why did you do this?

Reply:
So I could use that list in storybook